packages/lib/services/rest/routes/notes.ts

```diff
@@ -200,6 +200,13 @@ async function downloadMediaFile(url: string /* , allowFileProtocolImages */) {
 		return '';
 	}
 
+	const invalidProtocols = ['cid:'];
+	const urlProtocol = urlUtils.urlProtocol(url)?.toLowerCase();
```

Comment:
I don't think `?` is necessary, since it will return an empty string on failure?